File: stepik-java-api/src/main/java/org/stepik/api/actions/StepikStepsAction.java
package org.stepik.api.actions;

import org.jetbrains.annotations.NotNull;
import org.stepik.api.client.StepikApiClient;
import org.stepik.api.queries.steps.StepikStepsGetQuery;

/**
 * @author meanmail
 */
public class StepikStepsAction extends StepikAbstractAction {
    public StepikStepsAction(@NotNull StepikApiClient stepikApiClient) {
        super(stepikApiClient);
    }

    @NotNull
    public StepikStepsGetQuery get() {
        return new StepikStepsGetQuery(this);
    }
}

File: stepik-java-api/src/main/kotlin/org/stepik/api/actions/StepikStepsAction.kt
package org.stepik.api.actions

import org.stepik.api.client.StepikApiClient
import org.stepik.api.queries.steps.StepikStepsGetQuery

class StepikStepsAction(stepikApiClient: StepikApiClient) : StepikAbstractAction(stepikApiClient) {
    
    fun get(): StepikStepsGetQuery {
        return StepikStepsGetQuery(this)
    }
}
